# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: js/src/jit/loong64/Assembler-loong64.cpp
# Commit: cf96078694ae
# Full Hash: cf96078694aea74cae4557f683a76c6de6ba9cf1
# Author: Ryan Hunt <rhunt@eqrion.net>
# Date: 2023-08-05 09:19:01
# Regressor Bug: 1692065
# File Overlap Count: 3
# Description:
#   Bug 1692065 - wasm: Rename MIRType::RefOrNull to 'WasmAnyRef'. r=yury
#   
#   This is a mechanical change. The previous name 'RefOrNull' was too
#   generic, and it wasn't clear that the type was actually a wasm::AnyRef
#   type.
# ==============================================================================

diff -r 662cb63c42d7 -r cf96078694ae js/src/jit/loong64/Assembler-loong64.cpp
--- a/js/src/jit/loong64/Assembler-loong64.cpp	Fri Aug 04 14:06:49 2023 +0000
+++ b/js/src/jit/loong64/Assembler-loong64.cpp	Fri Aug 04 14:06:49 2023 +0000
@@ -32,7 +32,7 @@
     case MIRType::Int32:
     case MIRType::Int64:
     case MIRType::Pointer:
-    case MIRType::RefOrNull:
+    case MIRType::WasmAnyRef:
     case MIRType::StackResults: {
       if (intRegIndex_ == NumIntArgRegs) {
         current_ = ABIArg(stackOffset_);